dashboard/pages/home.py

```diff
@@ -13,7 +13,8 @@ def show_home():
 
     st.title("Home")
     st.header("Welcome to the Bandcamp analysis tracker!")
-    st.write("Here you'll find insights into Bandcamp sales and you can also subscribe to receive email notifications!")
+    st.write("""Here you'll find insights into Bandcamp sales
+             and you can also subscribe to receive email notifications!""")
```

Comment:
Double check this. I'm pretty sure when you use triple quotes, the indents will be included in the new line.

Reply:
🤨 I've tested the dashboard after making this change and it looks the same